tachyon/zk/lookup/halo2/permute_expression_pair.h

```diff
@@ -52,7 +52,7 @@ template <typename PCS, typename Evals, typename F = typename Evals::Field>
     }
   }
 
-  std::vector<F> permuted_table_expressions(domain_size);
+  std::vector<F> permuted_table_expressions(domain_size, F::Zero());
```

Comment:
It seems like loop below fills the values up to usable rows. just initialize the values for remaining indicies. But just add a comment that why this is safe.